docs/source/guides/upload.md

```diff
@@ -371,11 +376,89 @@ In addition to [`upload_file`] and [`upload_folder`], the following functions al
 
 For more detailed information, take a look at the [`HfApi`] reference.
 
-## Push files with Git LFS
+## Tips and tricks for large uploads
```

Comment:
Let's be careful to not break this url

Reply:
I would not be too worried about it to be honest. I've checked and `#upload-files-with-git-lfs` is referenced nowhere in our internal docs (both hfh and hf_docs). Doesn't mean that such a URL doesn't exist in the wild but I would expect probability to be quite low. And even if it's the case, users will be redirected to the correct page even though it's not the correct section. Since Git LFS upload is mostly deprecated _it should be fine_